src/js/cf-expandables.js

```diff
@@ -5,110 +5,238 @@
+      event.preventDefault();
+      event.stopPropagation();
+
+      if ( this.isExpanded ) {
+        this.collapse();
+      } else {
+        if ( this.isInAccordion ) {
+          this.$el.siblings( '.expandable' )
+          .each( function( index, sibling ) {
             sibling.collapse();
-          });
-        }
-      };
-
-      this.toggle = function() {
-        if ( $target.attr('aria-pressed') === 'true' ) {
-          this.collapse();
-        } else {
-          this.expand();
+          } );
         }
-      };
-
-      this.expand = function( duration ) {
-        $cueOpen.css( 'display', 'none' );
-        $cueClose.css( 'display', 'inline' );
-        $content.attr( 'aria-expanded', 'true' );
-        $target.attr( 'aria-pressed', 'true' );
-        if ( typeof duration === 'undefined' ) {
-          duration = $.fn.expandable.calculateExpandDuration( $content.height() );
-        }
-        $this.addClass('expandable__expanded');
-        $content.slideDown({
-          duration: duration,
-          easing: 'easeOutExpo'
-        });
-      };
-
-      this.collapse = function( duration ) {
-        $cueOpen.css( 'display', 'inline' );
-        $cueClose.css( 'display', 'none' );
-        $content.attr( 'aria-expanded', 'false' );
-        $target.attr( 'aria-pressed', 'false' );
-        if ( typeof duration === 'undefined' ) {
-          duration = $.fn.expandable.calculateCollapseDuration( $content.height() );
-        }
-        $this.removeClass('expandable__expanded');
-        $content.slideUp({
-          duration: duration,
-          easing: 'easeOutExpo'
-        });
-      };
-
-      this.init();
-
-    });
+        this.expand();
+      }
+    },
+
+    /**
+     * Expands the expandable, sets the internal state
+     * and adds aria attributes.
+     *
+     * @param {integer} duration -
+     *   The time duration in which the expansion will occur.
+     */
+    expand: function expand( duration ) {
+      var $content = this.$content;
+      this.$target.attr( 'aria-pressed', 'true' );
+      $content.attr( 'aria-expanded', 'true' );
+      if ( typeof duration === 'undefined' ) {
+        duration = $.fn.expandable.calculateExpandDuration( $content.height() );
+      }
+      this.$el.addClass( this.expandedClass );
+      $content.slideDown( {
+        duration: duration,
+        easing:   'easeOutExpo'
+      } );
+      this.isExpanded = true;
+    },
+
+    /**
+     * Collapses the expandable, sets the internal state
+     * and adds aria attributes.
+     *
+     * @param {integer} duration -
+     *   The time duration in which the collapse will occur.
+     */
+    collapse: function collapse( duration ) {
+      var $content = this.$content;
+      this.$target.attr( 'aria-pressed', 'false' );
+      $content.attr( 'aria-expanded', 'false' );
+      if ( typeof duration === 'undefined' ) {
+        duration = $.fn.expandable.calculateExpandDuration( $content.height() );
```

Comment:
oy vey, fixed in https://github.com/sebworks/cf-expandables/commit/9435e5e87dba6a6a1ebfa4d0428d77c6167e2481